app/LoginButton/index.js

```diff
@@ -15,6 +16,8 @@ const LoginButton = () => {
       console.log(`Code: ${code}`);
       const token = await getToken(code);
```

Comment:
```suggestion
      const token = await getToken();
```
Ya se que no es de tu pr, pero si podes cambiarlo :)